TombEngine/Game/effects/spark.cpp

```diff
@@ -87,7 +88,7 @@ namespace TEN::Effects::Spark
 			v += Vector3(GenerateFloat(-64, 64), GenerateFloat(-64, 64), GenerateFloat(-64, 64));
 			v.Normalize(v);
 			s.velocity = v * GenerateFloat(17, 24);
-			s.sourceColor = Vector4(1, 0.8f, 0.2f, 1) * 3;
+			s.sourceColor = colorStart * 3;
```

Comment:
Even if original issue, does it mean that color will be clipped above normal range?

Reply:
does i remove * 3 then ?

Reply:
I think you should compare visually how it looks with and without multiplier. The operation itself looks extremely weird and pointless indeed.

Reply:
Is there any update on this?

Reply:
it do the same, a bit less colorful but it don't change anything.

Reply:
removed